packages/attribute-registry-process/src/routers/AttributeRouter.ts

```diff
@@ -46,7 +49,8 @@ const attributeRegistryService = attributeRegistryServiceBuilder(
 );
 
 const attributeRouter = (
-  ctx: ZodiosContext
+  ctx: ZodiosContext,
+  service: AttributeRegistryService = attributeRegistryService
```

Comment:
Suggestion: name this `attributeRegistryService` and if needed rename the one above, to avoid the diff in all methods of the router below
```suggestion
  attributeRegistryService: AttributeRegistryService = defaultAttributeRegistryService
```

Reply:
Makes sense, done in 1faa33ee7a704588c50e5cf4b0b9fff62221ab69